src/ObjectLoader.cpp

```diff
@@ -0,0 +1,100 @@
+#include <cstring>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include <glm/glm.hpp>
+
+#include "ObjectLoader.h"
+#include "Mesh.h"
+
+/**
+ * Here is a short list of features a real function would provide : 
+ * 	- Binary files. Reading a model should be just a few memcpy's away, not parsing a file at runtime. In short : OBJ is not very great.
+ * 	- Animations & bones (includes bones weights)
+ * 	- Multiple UVs
+ * 	- All attributes should be optional, not "forced"
+ * 	- More stable. Change a line in the OBJ file and it crashes.
+ * 	- More secure. Change another line and you can inject code.
+ * 	- Loading from memory, stream, etc
+ */
+
+int loadObject(const char* path, 
+			   std::vector<glm::vec3>& out_vertices,
```

Comment:
I don't recommend trying to align stuff with tabs, it will show differently on everyone's editors. I think the best thing to do with tabs is just indent once in this circumstance.